package.json

```diff
@@ -7,7 +7,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "node-sass": "^7.0.3",
```

Comment:
is there any reason to change the version of sass? keep it same as the cloned version